src/templates/gwfcore/gwfcore-iam.template.yaml

```diff
@@ -33,6 +33,19 @@ Resources:
                 Resource: !Join ["", ["arn:aws:s3:::", !Ref S3BucketName, "/*"]]
                 Action:
                   - "s3:*"
+        - PolicyName: !Sub SecretManager-Access-${AWS::Region}
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - secretsmanager:GetSecretValue
+                Resource:
+                  - !Sub arn:aws:secretsmanager:*:${AWS::AccountId}:*
+              - Effect: Allow
+                Action:
+                  - secretsmanager:ListSecrets
+                Resource: "*"
```

Comment:
Same comment on scoping down to specific resources

Reply:
Same as before, ListSecrets still needs the wildcard